arches_templating/views/template.py

```diff
@@ -52,6 +52,13 @@ def post(self, request, templateid):
         bytestream = BytesIO()
         extension = os.path.splitext(template)[1].replace(".", "")
         factory = TemplateEngineFactory()
+
+        # I don't love doing this, because it's a "hardcoded" dependency on Arches - but it's optional.
+        try:
+            json_data['internal_base_url'] = settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT
```

Comment:
Since this is in the body of a view, could you just get the host from the request?
https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.get_host